spark/ClickBenchRunner.scala

```diff
@@ -0,0 +1,136 @@
+    StructField("IsOldCounter", ShortType, nullable = false),
+    StructField("IsEvent", ShortType, nullable = false),
+    StructField("IsParameter", ShortType, nullable = false),
+    StructField("DontCountHits", ShortType, nullable = false),
+    StructField("WithHash", ShortType, nullable = false),
+    StructField("HitColor", StringType, nullable = false),
+    StructField("LocalEventTime", TimestampType, nullable = false),
+    StructField("Age", ShortType, nullable = false),
+    StructField("Sex", ShortType, nullable = false),
+    StructField("Income", ShortType, nullable = false),
+    StructField("Interests", ShortType, nullable = false),
+    StructField("Robotness", ShortType, nullable = false),
+    StructField("RemoteIP", IntegerType, nullable = false),
+    StructField("WindowName", IntegerType, nullable = false),
+    StructField("OpenerName", IntegerType, nullable = false),
+    StructField("HistoryLength", ShortType, nullable = false),
+    StructField("BrowserLanguage", StringType, nullable = false),
+    StructField("BrowserCountry", StringType, nullable = false),
+    StructField("SocialNetwork", StringType, nullable = false),
+    StructField("SocialAction", StringType, nullable = false),
+    StructField("HTTPError", ShortType, nullable = false),
+    StructField("SendTiming", IntegerType, nullable = false),
+    StructField("DNSTiming", IntegerType, nullable = false),
+    StructField("ConnectTiming", IntegerType, nullable = false),
+    StructField("ResponseStartTiming", IntegerType, nullable = false),
+    StructField("ResponseEndTiming", IntegerType, nullable = false),
+    StructField("FetchTiming", IntegerType, nullable = false),
+    StructField("SocialSourceNetworkID", ShortType, nullable = false),
+    StructField("SocialSourcePage", StringType, nullable = false),
+    StructField("ParamPrice", LongType, nullable = false),
+    StructField("ParamOrderID", StringType, nullable = false),
+    StructField("ParamCurrency", StringType, nullable = false),
+    StructField("ParamCurrencyID", ShortType, nullable = false),
+    StructField("OpenstatServiceName", StringType, nullable = false),
+    StructField("OpenstatCampaignID", StringType, nullable = false),
+    StructField("OpenstatAdID", StringType, nullable = false),
+    StructField("OpenstatSourceID", StringType, nullable = false),
+    StructField("UTMSource", StringType, nullable = false),
+    StructField("UTMMedium", StringType, nullable = false),
+    StructField("UTMCampaign", StringType, nullable = false),
+    StructField("UTMContent", StringType, nullable = false),
+    StructField("UTMTerm", StringType, nullable = false),
+    StructField("FromTag", StringType, nullable = false),
+    StructField("HasGCLID", ShortType, nullable = false),
+    StructField("RefererHash", LongType, nullable = false),
+    StructField("URLHash", LongType, nullable = false),
+    StructField("CLID", IntegerType, nullable = false))
+)
```

Comment:
there is no way to create an index, right?

Reply:
Yes, no index support
https://spark.apache.org/docs/1.4.0/api/java/org/apache/spark/sql/types/StructField.html